### ISSUE TITLE
Custom covmat tutorial points to a non existing file

### ISSUE THREAD

**Issue by Zaharid** [NNPDF/nnpdf#1577]
The tutorial here

https://docs.nnpdf.science/tutorials/general_th_covmat.html

points to an non existing file

https://vp.nnpdf.science/IeGM9CY8RxGcb5r6bIEYlQ==/shrek_covmat.csv

which makes it rather hard to follow or understand much.

The report itself does exist

https://vp.nnpdf.science/IeGM9CY8RxGcb5r6bIEYlQ==/

so I am wondering if it is as simple as replacing the reference for one of the files there.

**Comment by enocera** [comment]
Thanks for noticing that. Let me also check with Rosalyn.

**Comment by andreab1997** [comment]
I believe that most likely the name of the file is just changed from _shrek_covmat.csv_ to _topthcovmat.csv_. However I believe that this is just an example regarding the address so just changing the name of the file should be ok.
